### ISSUE TITLE
LOR/Superstar LMS vs SUP color issues

### ISSUE THREAD

**Issue by nmiller0113** [smeighan/xLights#2367]
NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Description, reproduction, and expectation**
I've not done this in over a year as I've not had a new LOR/Superstar sequence to import since then, but it appears something isn't working properly now. I've always chosen to import and map my sequences from and LMS file, sourced from an SUP, as the SUP imports aren't 1:1 with the images and effects. Despite the bloated files due to all the channel data and the extended loading times, I appreciate the accuracy of the details as they were made in the Superstar Sequencer...so it's worth it to me. Plus, I only need to go through it once as I'm not editing it afterward.

I've done this successfully for years but today when I imported the LMS file I noticed that I could barely see the colors...they were dim and very faded...some weren't even correct. I had never seen this issue in the past. I wanted to confirm it wasn't the file I was importing from and did a standard SUP file import and the colors looked bright, vivid and accurate. I believe that something isn't working when importing the color intensities from an LMS file and I'm hoping you can possibly figure out the problem.

**Screenshots**
Happy to provide a video or screenshots of me reproducing this, but I'm not sure it's necessary. Also happy to jump on a zoom

**Versions (please complete the following information):**
 - OS: Mac 10.15.7
 - xLights version 2020.48.1

**Attachments**
Keith and Dan, since I've got your email address I've shared a [Google Drive Folder](https://drive.google.com/drive/folders/1kD5lHRwQ8nNIdUsVL6T4FWrbiRqytu2K?usp=sharing) with you containing both the LMS and SUP file it was exported from. I can share it with others as needed too. Thank you all for your assistance.

**Comment by nmiller0113** [comment]
I did some additional testing today and this also happens in Windows, so it's not something specific to Mac.

I checked the intensity levels in the LMS file and they look correct, but I noticed that xLights doesn't work the same, so I couldn't identify exactly where the difference was or why the colors don't match, but I'm assuming it's because xLights scales intensity differently. Hopefully, it can be corrected as the Superstar import colors look correct. Thanks again!

**Comment by keithsw1111** [comment]
I spent some time on this and see the problem but not why ... and not why now. Due to data size this is going to be a pain to diagnose. Its a pity it isnt a 3x4 tree instead of 16x50.

**Comment by nmiller0113** [comment]
@keithsw1111 I appreciate the work and help. I discussed this with @dkulp over email and he pointed out that the intensity levels in the LMS file were also incorrect/low and suspected it was something in the SUP to LMS export process. I looked further and it appears a default setting changed in Superstar where the export was applying color curves rather than the raw color values. I found a preferences setting in Superstar called "Export Raw Color Values" and checking this box corrected the export which also translated properly on the import into xLights. My apologies for any time you spent on this false bug report that turned out to be on the Superstar side and not xLights related!
